rust/src/cliwrap/kernel_install.rs

```diff
@@ -64,13 +67,20 @@ fn redo_systemctl_wrap() -> Result<()> {
     Ok(())
 }
 
-#[context("Running dracut")]
-fn run_dracut(kernel_dir: &str) -> Result<()> {
+#[context("Generate initfs")]
+fn generate_initfs(kernel_dir: &str) -> Result<()> {
```

Comment:
Sorry I should have mentioned this earlier but you have found a pre-existing confusing mess because we have *two* places that we run dracut, one here and the other in `rpmostree-kernel.cxx`.  I am pretty sure what you really want is to modify `rpmostree-kernel.cxx` because it's the one that runs when creating a "base image" aka new commit.  This code is trying to wrap the `dracut` command when invoked on the client side (which can be convenient for testing I guess).